playwright/specs/onboarding.spec.ts

```diff
@@ -3,45 +3,145 @@ import { expect } from "@playwright/test";
+    });
+
+    await test.step("Go through Basics Carrousel", async () => {
+      expect(await page.screenshot()).toMatchSnapshot(`carrousel-1.png`);
+      await onboardingPage.basicsCarrouselRight();
+      expect(await page.screenshot()).toMatchSnapshot(`carrousel-2.png`);
+      await onboardingPage.basicsCarrouselRight();
+      expect(await page.screenshot()).toMatchSnapshot(`carrousel-3.png`);
+      await onboardingPage.basicsCarrouselRight();
+      expect(await page.screenshot()).toMatchSnapshot(`carrousel-4.png`);
+      await onboardingPage.basicsCarrouselRight();
+      expect(await page.screenshot()).toMatchSnapshot(`carrousel-5.png`);
+      await onboardingPage.setupWallet();
+      expect(await page.screenshot()).toMatchSnapshot(`setup-wallet`);
+    });
+
+    await test.step("Set Up new device", async () => {
+      expect(await page.screenshot()).toMatchSnapshot(`start-setup.png`);
+      await onboardingPage.startSetup();
+      expect(await page.screenshot()).toMatchSnapshot(`set-pincode.png`);
+      await onboardingPage.setPincode();
+      expect(await page.screenshot()).toMatchSnapshot(`set-passphrase.png`);
+      await onboardingPage.setPassphrase();
+    });
+
+    await test.step("Pass Quiz", async () => {
+      expect(await page.screenshot()).toMatchSnapshot(`start-quiz.png`);
+      await onboardingPage.startQuiz();
+      expect(await page.screenshot()).toMatchSnapshot(`quiz-first-question.png`);
+      await onboardingPage.answerQuizBottom();
+      expect(await page.screenshot()).toMatchSnapshot(`quiz-next-question.png`);
+      await onboardingPage.quizNextQuestion();
+      expect(await page.screenshot()).toMatchSnapshot(`quiz-second-question.png`);
+      await onboardingPage.answerQuizBottom();
+      expect(await page.screenshot()).toMatchSnapshot(`quiz-next-question-2.png`);
+      await onboardingPage.quizNextQuestion();
+      expect(await page.screenshot()).toMatchSnapshot(`quiz-third-question.png`);
+      await onboardingPage.answerQuizTop();
+      expect(await page.screenshot()).toMatchSnapshot(`quiz-next-question-3.png`);
+      await onboardingPage.quizNextQuestion();
+      expect(await page.screenshot()).toMatchSnapshot(`quiz-success.png`);
+      await onboardingPage.quizEnd();
+    });
+
+    await test.step(`[${nano}]"Device genuine check"`, async () => {
+       expect(await page.screenshot()).toMatchSnapshot(`connect-${nano}-new.png`);
+       await onboardingPage.checkDevice();
+       expect(await page.screenshot()).toMatchSnapshot(`before-genuine-check-new.png`);
+     });
+
+
+    await test.step("Pass genuine check", async () => {
+      await deviceAction.genuineCheck();
+      expect(await page.screenshot()).toMatchSnapshot("genuine-check-done-new.png");
+    });
+
+    await test.step("Reach app", async () => {
+      await onboardingPage.continue();
+      expect(await page.screenshot()).toMatchSnapshot(`onboarding-complete-new.png`);
+    });
+
+  }); 
+  }
+});
```

Comment:
Very small thing: try to sort the brackets out. The easy way is to have the Prettier extension installed in VSCode and set it up to apply prettier fixes when you save https://scottsauber.com/2017/06/10/prettier-format-on-save-never-worry-about-formatting-javascript-again/